metadata.mk

```diff
@@ -13,6 +13,7 @@ KUBECTL_VERSION = v1.22.1
 COREDNS_VERSION=1.5.2
 ETCD_VERSION=v3.3.7
 PROTOC_VER=v0.1
+UBI_VERSION=8.5-218
```

Comment:
I think it's OK to let this float at 8.5 - we'll pick up some small bugfixes / CVE fixes without risk of major changes slipping in.

Reply:
got it, makes sense. will fix!